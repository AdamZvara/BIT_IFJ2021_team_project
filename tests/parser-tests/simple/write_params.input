require "ifj21"

function main(num : integer, double : number, text : string)
	write(num," ",double," ", text, "\n")
end

main(1,1.1,"Test")
