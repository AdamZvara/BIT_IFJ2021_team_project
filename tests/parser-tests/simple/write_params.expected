1 0x1.199999999999ap+0 Test
